frontend/express/public/localization/dashboard/dashboard.properties

```diff
@@ -17,17 +17,11 @@ common.session-duration = Session Duration
 common.table.no-data = No data
 common.search.no-match-found = No match found
 common.table.total-users = Total Users
-common.table.total-users-desc = Number of times your application is opened by a new or returning user from a unique device, in the selected time period.
```

Comment:
Please don't delete/overwrite localization strings !!!!

Reply:
okay, please be calm.